### ISSUE TITLE
There is no Place card on the top of the first trip. version 1.2.2

### ISSUE THREAD

**Issue by MaliheTabasi** [e-mission/e-mission-docs#884]
@shankari There is no Place card on the top of the first trip. This Place card will be needed to insert activities before the first trip. Please see screenshot below.
![msg-908960961-663808](https://user-images.githubusercontent.com/49550315/230753810-699e510d-4fef-4491-94db-d6768a601548.jpg)

**Comment by shankari** [comment]
@MaliheTabasi @sebastianbarry is working on this change. It is not expected to be complete yet.
https://github.com/e-mission/e-mission-phone/pull/939

There are also open questions around multi-day places that we should finalize before this new requested feature is done.

Note that we did not discuss this "first place" feature in the original design sketch

And I did not indicate that the feature was done in 
https://github.com/e-mission/e-mission-docs/issues/871#issuecomment-1498520572

which is why it is useful to file separate issues for each new feature.
I'm glad to see you have done so, and we will track this request in this issue going forward.
